addons/base_import/static/src/js/import.js

```diff
@@ -217,24 +256,70 @@ var DataImport = Widget.extend(ControlPanelMixin, {
             },
         });
     },
+    setup_float_format_picker: function () {
+        var sug_query = function (q) {
+                    var suggestions = [
+                        {id: ',', text: _t("Comma")},
+                        {id: '.', text: _t("Dot")},
+                    ];
+                    if (q.term) {
+                        suggestions.unshift({id: q.term, text: q.term});
+                    }
+                    q.callback({results: suggestions});
+                };
+        this.$('input.oe_import_float_thousand_separator').select2({
+            width: '160px',
+            query: sug_query,
+            initSelection: function (e, c) {
+                return c({id: ',', text: _t("Comma")});
+            },
+        });
+        this.$('input.oe_import_float_decimal_separator').select2({
+            width: '160px',
+            query: sug_query,
+            initSelection: function (e, c) {
+                return c({id: ',', text: _t("Dot")});
+            },
+        });
+    },
 
     import_options: function () {
         var self = this;
         var options = {
-            headers: this.$('input.oe_import_has_header').prop('checked')
+            headers: this.$('input.oe_import_has_header').prop('checked'),
+            advanced: this.$('input.oe_import_advanced_mode').prop('checked'),
+            keep_matches: this.do_not_change_match
         };
         _(this.opts).each(function (opt) {
             options[opt.name] =
                 self.$('input.oe_import_' + opt.name).val();
         });
+        _(this.parse_opts).each(function (opt) {
+            if (opt.name === 'date_format') {
+                options[opt.name] = time.moment_to_strftime_format(self.$('input.oe_import_date_format').val());
+            }
+            else {
+                options[opt.name] = self.$('input.oe_import_' + opt.name).val();
+            }
+        });
+        options['fields'] = [];
+        if (this.do_not_change_match) {
+            options['fields'] = this.$('.oe_import_fields input.oe_import_match_field').map(function (index, el) {
+                return $(el).select2('val') || false;
+            }).get();
+        }
+        this.do_not_change_match = false;
         return options;
     },
 
     //- File & settings change section
     onfile_loaded: function () {
+        var file = this.$('.oe_import_file')[0].files[0];
```

Comment:
Not sure that actually works on IE9 or cell phones: http://caniuse.com/#feat=input-file-multiple